binnacle/lib/main.dart

```diff
@@ -47,8 +49,9 @@ class MyHomePage extends StatefulWidget {
 }
 
 class _MyHomePageState extends State<MyHomePage> {
-  int _counter = 0;
   double _direction;
+  var _counter;
+  var _acceleroList = new List(3);
```

Comment:
Might be better to store the _acceleroList as an AccelerometerEvent for readability when using the variable.